app/services/recovery_code_generator.rb

```diff
@@ -1,7 +1,7 @@
 class RecoveryCodeGenerator
   attr_reader :user_access_key
 
-  def initialize(user, length: 5)
+  def initialize(user, length: 4)
```

Comment:
the full background is in the thread in https://github.com/18F/identity-private/issues/1564

tl;dr is, we want 16 characters in 4 groups of 4 characters each.